actors/evm/src/state.rs

```diff
@@ -17,6 +17,26 @@ pub struct Tombstone {
     pub nonce: u64,
 }
 
+/// A structure representing Transient Data
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Serialize_tuple, Deserialize_tuple)]
+pub struct TransientData {
+    /// The contract transient data state dictionary.
+    /// Transient Data State is a map of U256 -> U256 values.
+    /// KAMT<U256, U256>
+    pub transient_data_state: Option<Cid>,
+    /// The data representing the transient data lifespan
+    pub transient_data_lifespan: Option<TransientDataLifespan>,
```

Comment:
No need for either of these to be options.

Reply:
that's helpful, thanks! 978b49d1e0f1e50fbecf020cadcfb1b5b8c4275a